Cant update or re-install

When trying to update (or re-install when update failed) I get the following error:

```
Installing “atom-ternjs@0.7.1” failed.Hide output…

npm ERR! git fetch -a origin (https://github.com/marijnh/tern.git) fatal: Unable to find remote helper for 'https'
npm ERR! git fetch -a origin (https://github.com/angelozerr/tern-lint.git) fatal: Unable to find remote helper for 'https'
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "c:\\Users\\wmrawlin.SBIZ\\AppData\\Local\\atom\\app-1.0.7\\resources\\app\\apm\\bin\\\\node.exe" "c:\\Users\\wmrawlin.SBIZ\\AppData\\Local\\atom\\app-1.0.7\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\wmrawlin.SBIZ\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\wmrawlin.SBIZ\\.atom\\.apmrc" "install" "C:\\Users\\WMRAWL~1.SBI\\AppData\\Local\\Temp\\d-115714-14772-1l6j6er\\package.tgz" "--target=0.22.0" "--arch=ia32" "--msvs_version=2012"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code 128

npm ERR! Command failed: fatal: Unable to find remote helper for 'https'
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>
```

I'm not entirely sure if this is a problem with my npm install or with atom or with the plugin.

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
   "dependencies": {
     "jquery": "^2.1.4",
     "loophole": "^1.1.0",
-    "tern": "^0.15.0",
+    "tern": "marijnh/tern",
     "tern-lint": "^0.5.0",
     "underscore-plus": "^1.6.6"
   },

--- a/package.json
+++ b/package.json
@@ -11,11 +11,11 @@
     "atom": ">=1.0.0"
   },
   "dependencies": {
-    "jquery": "*",
-    "loophole": "*",
-    "tern": "git+https://github.com/marijnh/tern.git",
-    "tern-lint": "git+https://github.com/angelozerr/tern-lint.git",
-    "underscore-plus": "*"
+    "jquery": "^2.1.4",
+    "loophole": "^1.1.0",
+    "tern": "^0.15.0",
+    "tern-lint": "^0.5.0",
+    "underscore-plus": "^1.6.6"
   },
   "providedServices": {
     "autocomplete.provider": {

